core/esmf-aspect-meta-model-python/README.md

```diff
@@ -92,3 +149,78 @@ units_graph.print_description(unit_data)
 # ...
 # symbol: V
 ```
+
+## SAMM CLI wrapper
+
+The SAMM CLI is a command line tool provided number of functions for working with Aspect Models.
+
+ore detailed information about SAMM CLI functionality can be found in the 
```

Comment:
This paragraph is missing the start of a sentence

Reply:
Fixed